src/Castle.Core.Tests.WeakNamed/Castle.Core.Tests.WeakNamed.csproj

```diff
@@ -3,7 +3,7 @@
 	<Import Project="..\..\buildscripts\common.props"></Import>
 
 	<PropertyGroup>
-		<TargetFrameworks>net461;netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks>net461;netcoreapp2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
(This is fine! Even if we *don't* add a test run for .NET Core 2.0 to our CI builds just yet, I think it's a good idea to still build for that platform to ensure that compilation for .NET Standard 2.0 actually succeeds.)